discourse_migration/userguide.md

```diff
@@ -45,48 +45,49 @@ are the typical use cases:
 
 This table explains the mapping from mailing lists to categories in Discourse.
 The email addresses of these categories will remain the same, after the
+migration.  Deprecated lists will become read-only as part of the Discourse
 migration.
 
 | Mailing lists         | category in Discourse |
 |-----------------------|--------------------|
 | All-commits           | no migration at the moment |
 | Bugs-admin            | no migration at the moment |
 | cfe-commits           | no migration at the moment |
-| cfe-dev               | llvm-project/clang |
-| cfe-users             | **TODO** llvm-project/clang or create new category? |
-| clangd-dev            | llvm-project/clangd |
-| devmtg-organizers     | **TODO** create new category |
+| cfe-dev               | Clang Compiler & Tools |
+| cfe-users             | Clang Compiler & Tools/Using Clang |
+| clangd-dev            | Clang Compiler & Tools/clangd |
+| devmtg-organizers     | Deprecated |
 | Docs                  | infrastructure/documentation |
-| eurollvm-organizers   | **TODO** create new category |
+| eurollvm-organizers   | Deprecated |
 | flang-commits         | no migration at the moment |
-| flang-dev             | llvm-project/flang |
+| flang-dev             | LLVM Based Subprojects/flang |
 | gsoc                  | community/gsoc |
 | libc-commits          | no migration at the moment |
-| libc-dev              | llvm-project/libc |
-| Libclc-dev            | **TODO** create new category |
+| libc-dev              | Runtimes/libc |
+| Libclc-dev            | Runtimes/libclc |
 | libcxx-bugs           | no migration at the moment |
 | libcxx-commits        | no migration at the moment |
-| libcxx-dev            | llvm-project/libcxx |
+| libcxx-dev            | Runtimes/libcxx |
 | lldb-commits          | no migration at the moment |
-| lldb-dev              | llvm-project/lldb |
-| llvm-admin            | **TODO** create new category |
-| llvm-announce         | **TODO** create new category |
+| lldb-dev              | LLVM Based Subprojects/lldb |
+| llvm-admin            | Replace with bugzilla or Github Issues |
```

Comment:
Recommend "no migration at the moment" unless there is a specific plan already.

Reply:
No migration for LLDB?

Reply:
This is proposing LLDB go into "Subprojects / LLDB"